docs/source/execute_api.rst

```diff
@@ -0,0 +1,143 @@
+Executing notebooks
+===================
```

Comment:
Hmm, I would disagree with you there. I think executing and converting are two separate steps, and if all you want to do is execute a notebook, you shouldn't need an exporter object involved. So I would consider using `ExecutePreprocessor` the more canonical approach.